docs/api.json

```diff
@@ -0,0 +1,1070 @@
+      "POST": {
+        "description": "Adds a visit for a visitor",
+        "auth": true,
+        "intendedFor": ["visitor"],
+        "scope": ["visit_logs-parent:write"],
+        "body": {
+          "userId": "Number",
+          "activityId": "Number"
+        },
+        "response": "types.visit"
+      }
+      },
+      "/:id/volunteer_logs": {
+        "GET": {
+          "description": "Retreive volunteer information for a user",
+          "auth": true,
+          "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin", "volunteer"],
+          "scope": ["volunteer_logs-child:read", "volunteer_logs-own:read"],
+          "query": {
+            "fields": ["String"],
+            "filter": {
+              "since": "String",
+              "until": "String"
+            }
+          },
+          "response": {
+            "totalVolunteerHours": "Number",
+            "volunteerLogs": ["types.volunteerLog"]
+          }
+        }
+      },
+      "/:me/volunteer_logs": {
+        "POST": {
+          "description": "Add volunteer log for your user account",
+          "auth": true,
+          "intendedFor": ["volunteer"],
+          "scope": ["volunteer_logs-parent:write"],
+          "query": {
+            "organisationId":"Number",
+            "volunteerActvity": "String",
+            "duration": "Number",
+            "loggedAt": "String"
+          },
+          "response": "types.volunteerLog"
+        }
+      },
+      "/me/volunteer_logs/:id": {
+        "PUT": {
+          "description": "Update volunteer log for your user account",
+          "auth": true,
+          "intendedFor": ["volunteer"],
+          "scope": ["volunteer_logs-parent:write"],
+          "body": {
+            "organisationId":"Number",
+            "volunteerActvity": "String",
+            "duration": "Number",
+            "loggedAt": "String"
+          },
+          "response": "types.volunteerLog"
+        },
+        "DELETE": {
+          "description": "Delete volunteer log for your user account",
+          "auth": true,
+          "intendedFor": ["volunteer"],
+          "scope": ["volunteer_logs-parent:delete"],
+          "body": {
+            "organisationId":"Number",
+            "volunteerActvity": "String",
+            "duration": "Number",
+            "loggedAt": "String",
+            "delete": "Boolean"
+          },
+          "response": null
+        }
+      },
+      "/:id/outreach": {
+        "GET": {
+          "description": "Retrieve all outreach campaigns and meetings for a specific user",
+          "auth": true,
+          "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin", "volunteerAdmin"],
+          "scope": ["organisations_outreach-children:read", "organisations_outreach-own:read"],
+          "query": {
+            "fields": ["String"],
+            "filter": {
+              "outreachType": "String"
+            }
+          },
+          "response": {
+            "result": ["types.outreach"]
```

Comment:
Again, probs best to split this as described (way) above into `outreach_campaings` and `meetings`